### ISSUE TITLE
@Size on method parameter causes the model parsing to fail

### ISSUE THREAD

**Issue by callenx** [teamcarma/swagger-jaxrs-doclet#57]
Similar to #53, annotating @Size on a method's String parameter causes model parsing to fail.

Interface:

    @GET
    @Path("/product/{sku}")
    Product getProduct(@PathParam("sku") @NotNull @Size(min=10, max=10) String sku);

Produces:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.1:javadoc (generate-service-docs) on project online-rest-api: An error has occurred in JavaDocs report generation:
[ERROR] Exit code: 1 - javadoc: error - In doclet class com.carma.swagger.doclet.ServiceDoclet,  method start has thrown an exception java.lang.reflect.InvocationTargetException
[ERROR] java.lang.IllegalStateException: Min/Max values are not allowed for the type: string and format: null for the method: getProduct parameter: sku min value.
[ERROR] at com.carma.swagger.doclet.parser.ParserHelper.verifyNumericValue(ParserHelper.java:237)
[ERROR] at com.carma.swagger.doclet.parser.ApiMethodParser.generateParameters(ApiMethodParser.java:609)
...
```

**Comment by conorroche** [comment]
ive fixed this in the latest snapshot

**Comment by callenx** [comment]
Confirmed fixed in 1.0.5-SNAPSHOT.  You rock.

**Comment by conorroche** [comment]
@callenx ive just released 1.0.4.2 that contains this fix

**Comment by callenx** [comment]
I feel like it's my birthday.  : )
